isis/tests/FunctionalTestsIsisImportEis.cpp

```diff
@@ -0,0 +1,375 @@
+#include <iostream>
+#include <time.h>
+
+#include <QRegExp>
+#include <QString>
+#include <nlohmann/json.hpp>
+
+#include "Fixtures.h"
+#include "Histogram.h"
+#include "md5wrapper.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "PvlKeyword.h"
+#include "TestUtilities.h"
+
+#include "isisimport.h"
+
+#include "gmock/gmock.h"
+
+using namespace Isis;
+using namespace testing;
+using json = nlohmann::json;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/isisimport.xml").expanded();
+
+TEST_F(TempTestingFiles, FunctionalTestIsisImportEisNacFrame){
+  std::istringstream PvlInput(R"(
+    Object = IsisCube
+      Object = Core
+        StartByte   = 65537
+        Format      = Tile
+        TileSamples = 445
+        TileLines   = 958
+
+        Group = Dimensions
+          Samples = 1335
+          Lines   = 3832
+          Bands   = 1
+        End_Group
+
+        Group = Pixels
+          Type       = SignedWord
+          ByteOrder  = Lsb
+          Base       = 0.0
+          Multiplier = 1.0
+        End_Group
+      End_Object
+
+      Group = Instrument
+        SpacecraftName   = "Europa Clipper"
+        InstrumentId     = "EIS NAC PB"
+        TargetName       = Didymos
+        StartTime        = 2021-03-06T04:05:27.77
+        ExposureDuration = 1.0 <seconds>
+      End_Group
+
+      Group = BandBin
+        FilterName = CLEAR
+        Center     = 702.5 <nm>
+        Width      = 695 <nm>
+      End_Group
+
+      Group = Kernels
+        NaifFrameCode = -159103
+      End_Group
+    End_Object
+
+    Object = Label
+      Bytes = 65536
+    End_Object
+
+    Object = History
+      Name      = IsisCube
+      StartByte = 10296977
+      Bytes     = 657
+    End_Object
+
+    Object = OriginalXmlLabel
+      Name      = IsisCube
+      StartByte = 10297634
+      Bytes     = 13238
+      ByteOrder = Lsb
+    End_Object
+    End
+  )");
+  QString dataFileName = " data/eis2isis/nacFrame/nac000xxx_2022145t000000_0000000001_frame_raw02.xml";
```

Comment:
Shouldn't these files be in a dir named isisimport rather than eis2isis? Hopefully eis2isis will go away in the near future.

Reply:
I agree, we should call the directory something more generic. Maybe EISPDS